### DISCUSSION TITLE
T2 Ubuntu on Macbook Pro 15" 2018 with faulty AMD GPU

### DISCUSSION THREAD

**Discussion by Alf-Decondrius** [AdityaGarg8/mbp-16.x-ubuntu-kernel#66] [Q&A]
Hi there,
I have a Macbook Pro from 2018, with a faulty AMD GPU. 
As with many other 15" MBPs of 2018 (it seems to be a common issue since 2023. Feels a bit like programmed obsolescence), my MBP crashes when the AMD GPU kicks in, or sometimes even happens when you plug anything into the USBC because it seems to try to check if it is a monitor to route to the discrete GPU.
It happens on MacOS very frequently, but sometimes also on T2 Ubuntu.

My best guess would be to use it as an Ubuntu Server, because I had no luck trying to remove AMD drivers (or maybe I just did it the wrong way).

Does anyone have any idea what would be the best way to either make it work with Intel GPU only or go with an Ubuntu Server solution with no GUI?
If the Ubuntu Server is the only way, is there already a T2 Ubuntu Server version?

Thanks a lot!

**Comment by AdityaGarg8** [comment]
You can add `apple-gmux.force_igd` to your command line
